src/coffee-script.coffee

```diff
@@ -19,6 +19,12 @@ exports.FILE_EXTENSIONS = ['.coffee', '.litcoffee', '.coffee.md']
 # Expose helpers for testing.
 exports.helpers = helpers
 
+# Function that allows for btoa in both nodejs and the browser.
+base64encode = (src) ->
+  if typeof Buffer is 'function' then new Buffer(src).toString('base64')
+  else if typeof btoa is 'function' then btoa(src)
+  else throw new Error('Unable to base64 encode inline sourcemap.')
```

Comment:
Ah, I often do this. It's because originally there were only an if and an else. I'll get to refactoring this later in the day.